lms/envs/common.py

```diff
@@ -349,11 +349,8 @@
     # Show Language selector.
     'SHOW_LANGUAGE_SELECTOR': False,
 
-    # Write new CSM history to the extended table.
-    # This will eventually default to True and may be
-    # removed since all installs should have the separate
-    # extended history table.
-    'ENABLE_CSMH_EXTENDED': False,
+    # This may be removed when all installs have the separate extended history table.
```

Comment:
If people have options about how to handle the extended table, then we'll never be able to say, "all installs have the separate table", right?